README.md

```diff
@@ -1,2 +1,36 @@
 # back8sup
+
 A simple way to backup your kubernetes resources
+
+
+## Deployment w/ HELM v3.x
+
+```
+helm install back8sup charts/back8sup
+```
+## Configuration
+
+Configuration is done via environment variables inside the container and an additional configmap
+
+
+### Environment variables
+
+```
+API_ENDPOINT=${API_ENDPOINT:-https://kubernetes.local:6443}
+CA_CERT=${CA_CERT:-/etc/ssl/ca.crt}
+TOKEN_FILE=${TOKEN_FILE:-/var/run/secrets/sa}
+DST_FOLDER=${DST_FOLDER:-/mnt/back8sup}
+CONFIGMAP_PATH=${CONFIGMAP_PATH:-/etc/config.yaml}
+EXPORT_FORMAT=${EXPORT_FORMAT:-yaml}
+```
+
+### ConfigMap
+
```

Comment:
This should probably mention that the configmap is managed by Helm (At least I'm assuming it will be).

Reply:
```suggestion
(managed by Helm)
```

Reply:
I'm a fan of full sentences.

Reply:
better?